packages/core/react/src/__tests__/WalletProviderMobile-test.tsx

```diff
@@ -233,7 +233,7 @@ describe('WalletProvider when the environment is `MOBILE_WEB`', () => {
     });
     describe('when there exists no stored wallet name', () => {
         beforeEach(() => {
-            (localStorage.getItem as jest.Mock).mockReturnValue(null);
```

Comment:
These mocks never unhook. Much cleaner to just call `removeItem` or `setItem` to massage the cache into the state you want for the test, to later be cleared by `localStorage.clear()` in the top-level `beforeEach()` hook.